src/app/partials/nav/header-static/header-static.component.html

```diff
@@ -1,25 +1,17 @@
 <header id="header-static">
-  <ul class="inline-list left-list left">
-    <li class="logo">
-      <img src="/assets/images/EvalAI_favicon.png" alt="">
-    </li>
-  </ul>
-  <ul class="inline-list right-list right right-nav">
-    <li>
-      <a class="light-link" href="/"> Challenges</a>
-    </li>
-    <li>
-      <a class="light-link" href="/"> News</a>
-    </li>
-    <li>
-      <a class="light-link" href="/"> Forum</a>
-    </li>
-    <li>
-      <a class="light-link" href="/"> Join Us</a>
-    </li>
-    <li class="pointer">
-      ...
-    </li>
-  </ul>
-  <div class="clear-fix"></div>
+    <div class="logo_container">
+      <img src="assets/images/EvalAI_favicon.png" class = "logo_header" [class.invert_color]="globals.scrolledState"/>
+    </div>
+    <div class="navlinks_container" >
+    <div class="main-wrapper clearfix">
+        <span class = "navlinks navlinktext" [class.dark_text]="globals.scrolledState"> Challenges </span>
```

Comment:
Any reason why you have changed these lists to span?

Reply:
I can change them back to list elements. Should I ? I just used the header that I'd already built before 👱